### DISCUSSION TITLE
Version 10

### DISCUSSION THREAD

**Discussion by ClemensInBerlin** [fluentribbon/Fluent.Ribbon#1039] [General]
Hello, is there a release date for version 10?

**Comment by batzen** [comment]
Not yet.
I will have to wait for MahApps.Metro 3 to be released and the first preview of version 4 to be available, as quite a few people are using MahApps.Metro and Fluent.Ribbon in the same project (including myself).
As Fluent.Ribbon 10 depends on ControlzEx 6 it's incompatible with MahApps.Metro 2 and 3.
I already created a PR in MahApps.Metro to update ControlzEx there and fixed a few things with that yesterday (hopefully the last few things).
Please don't ask the same question in MahApps.Metro.
It's done when it's done. ;-)

While waiting for MahApps.Metro i will, most likely, do some code maintenance here (like finally renaming all resources to a common pattern etc.).
After MahApps.Metro 4 is available as a preview i can update my employers application and run the ~10 k automated UI tests, which tend to reveal if Fluent.Ribbon is stable or not.

I hope to release Fluent.Ribbon 10 in september.
